### PR TITLE
Final milestone: Styling -Finished Project

### PR DESCRIPTION
**Pull request by m05e5**
- [x] Finished all, including styling
- [x] Project's ready to merge
- [x] All functions are working as expected
image
image
image

Branch ready to merge, we are ready to submit for 

- [x] review.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,78 @@
...
+- Webpack 🤍
+
+---
+The main purpose of the project [redux_group_project]() is to put in practice HTML5, CSS3, and Javascript concepts to build a website with all the proffesional outview of it.
+~
+---
+# 🤍 Live Demo 🤍
+---
+
+🤍 [live demo]()
```
Comment (ZawadiSumba66) [inline README.md]: - [x] The live demo link to the project app is currently missing. Kindly fix it.
- [x] You ought to have added a video presentation for a clear and objective description of the project.

File: src/components/Header.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    <div className="logo-title-wrapper">
+      <img src={logo} alt="Logo" className="logo" />
+      <h1 className="nav-title">Space Travelers&#39;s Hub</h1>
+    </div>
+    <nav>
+      <div className="nav-div nav-list">
+        <ul className="navUl nav-rockets">
+          <li>
+            <Link to="/" className="navLink">Rockets</Link>
+          </li>
```
Comment (ZawadiSumba66) [inline src/components/Header.js:17]: - [x] currently active page should be indicated with an underline. Kindly lets fix this.

File: src/components/Missions.js [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+import { useDispatch, useSelector } from 'react-redux';
+import axios from 'axios';
+import Mission from './Mission';
+import './Missions.css';
+import { setMissions } from '../redux/reducers/MissionsReducer';
+
+const Missions = () => {
+  const missions = useSelector((state) => state.missionsReducer);
+  const dispatch = useDispatch();
+  console.log(missions);
```
Comment (ZawadiSumba66) [inline src/components/Missions.js]: - [x]  Kindly remove `console.log` statements from your file to make your code cleaner and neater.

File: src/redux/reducers/MissionsReducer.js [hunk 4]
```diff
@@ -0,0 +1,35 @@
+/* eslint-disable max-len */
```
Comment (ZawadiSumba66) [inline src/redux/reducers/MissionsReducer.js:1]: - [x] Let's ensure we try to fix the linters errors as much as possible by avoiding disabling them and instead try to find a solution which is a good practice as a professional developer.

**Review by ZawadiSumba66 — CHANGES_REQUESTED** [review]
### Hi  Team 👋 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
- [x] The store.js file is currently empty in the app folder. Kindly remove it if you do not intend to use it.
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (sheylaPozo) [reply README.md]: All checks from the review are `Completed! ` Thanks for the review. 😋✔👍😄

File: src/components/Header.js (hunk 2 above)
Comment (sheylaPozo) [reply src/components/Header.js:17]: oki fixed it :)

File: src/components/Missions.js (hunk 3 above)
Comment (sheylaPozo) [reply src/components/Missions.js]: okii fixed it :)

File: src/redux/reducers/MissionsReducer.js (hunk 4 above)
Comment (sheylaPozo) [reply src/redux/reducers/MissionsReducer.js:1]: fixed it shortening `"missions"` variable name into `"mis"`, `max-len` is 100 characters per line, so I had to :)

File: README.md (hunk 1 above)
Comment (sheylaPozo) [reply README.md]: Thanks for the review, my coding partner would prefer to do the video tomorrow, and that would be the last to add for approval. 👍☺

- [ ]

**Review by ZawadiSumba66 — APPROVED** [review]
Hi Team 😃 , 
### STATUS: APPROVED 👍 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by sheylaPozo** [comment]
Thank you for everything :)

**Pull request merged**